bayesian/utils.py

```diff
@@ -64,17 +64,19 @@ def get_user_email(user_profile):
 
 
 def create_component_bookkeeping(ecosystem, packages_list,
```

Comment:
Pass query params as an arg to create_component_bookkeeping to keep consistent with header arg

Reply:
Done.